example_configs/flume_agent.yaml

```diff
@@ -0,0 +1,39 @@
+# Config to expose JMX-variables from Flume's Java proccess and map to Prometheus syntax.
+# The FlumeKafkaSink metrics are also exposed.
+#
+# When using Hortonworks and Ambari. You can use the Ambari > Flume > flume-env and attach the java-agent with the config to the flume process.
+# Proposed directory: /etc/jmx_exporter/
+---
+lowercaseOutputName: true
+ 
+rules:
+- pattern: org.apache.flume.source<type=(.+)><>EventAcceptedCount
+  name: flume_source_accepted_count
+  labels: {"agent": "agent1", "source": $1}
```

Comment:
Yes, we have multiple agents, this is how we can distinguish them in Prometheus.
For the example I can leave them out.